src/app/products/product/product.component.html

```diff
@@ -0,0 +1,22 @@
+<div>
+  <span class="product__name">{{product.name}}</span>
+  <span class="product__description">{{product.description}}</span>
+  <span class="product__category">{{product.category}}</span>
+  <span class="product__ingredients">
+    <span *ngFor="let ingredient of product.ingredients; let isLast=last">
+      {{ingredient}}{{isLast ? '' : ', '}}
+    </span>
+  </span>
+  <span class="product__equivalents">
+    <span *ngFor="let equivalent of product.equivalents; let isLast=last">
+      {{getAlternative(equivalent).name}}{{isLast ? '' : ', '}}
+    </span>
+  </span>
+  <span class="product__availability">{{product.isAvailable ? '&#x2714;' : '&#x2716;'}}</span>
```

Comment:
Ideally, move in `config` or `characters` file

Reply:
maybe someday

Reply:
=)

Reply:
Smth like:

```jsx
const HEAVY_CHECK_MARK = '&#x2714;';
const HEAVY_CROSS = '&#x2716;';
<span class="product__availability">
    {{product.isAvailable 
        ? HEAVY_CHECK_MARK 
        : HEAVY_CROSS 
    }}
</span>
```

Reply:
Could be.